Assets/MRTK/Core/Inspectors/Profiles/MixedRealitySceneSystemProfileInspector.cs

```diff
@@ -19,11 +19,14 @@ public class MixedRealitySceneSystemProfileInspector : BaseMixedRealityToolkitCo
         const float DragAreaOffset = 10;
         const float LightingSceneTypesLabelWidth = 45;
 
+        private static string defaultSceneContent =
+            "Default scene system resources were not found.\nIf using custom manager and lighting scenes, this message can be ignored.\nIf not, please see the documentation for more information";
```

Comment:
Nit: Worth linking to documentation about where to find help?

Reply:
There is an existing documentation link as part of the inspector :)